libs/pmp-web/repository/repository-settings/feature/src/lib/containers/settings/common-settings/common-settings.component.scss

```diff
@@ -0,0 +1,25 @@
+.half-width-container {
```

Comment:
This class is created to later extend other classes with width and margin styles, but it is not used anywhere in the template.
A better practice would be to create a mixin instead of a class here.